### PR TITLE
refactor: dataCenter to cmp

### PR DESCRIPTION
**Pull request by laojun**
## What type of PR is this?

- [] Feature
- [ ] Bugfix
- [ ] Test
- [ ] Documentation
- [x] Refactor
- [ ] Style
- [ ] Chore

## What this PR does / why we need it:
refactor: dataCenter to cmp

## Which issue(s) this PR fixes:

Fixes #

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [ ] No

### CODE REVIEW

File: nginx.conf.template [hunk 1]
```diff
@@ -72,6 +72,11 @@ server {
         rewrite ^/([^/]*)/microService/(.*) $my_scheme://$http_host/$1/msp/$2 redirect; 
     }
 
+    # 301 redirect url with replace dataCenter to mcp
+    location ~ ^/[^/]*/microService/ {
```
Comment (daskyrk) [inline nginx.conf.template:76]: dataCenter, not microService

File: nginx.conf.template [hunk 2]
```diff
@@ -72,6 +72,11 @@ server {
...
+        rewrite ^/([^/]*)/dataCenter/(.*) $my_scheme://$http_host/$1/mcp/$2 redirect;
```
Comment (daskyrk) [inline nginx.conf.template:77]: cmp, not mcp

File: shell/app/layout/pages/page-container/components/sidebar.tsx [hunk 3]
```diff
@@ -53,10 +53,10 @@ const AppCenterEl = () => {
 
   const openMap = {
     orgCenter: permMap.entryOrgCenter.pass,
-    dataCenter: permMap.dataCenter.showApp.pass,
+    cmp: permMap.cmp.showApp.pass,
     dop: permMap.dop.read.pass,
     [dataEngineerInfo.name]: permMap.entryFastData.pass && currentOrg.openFdp, // this is backend attribute can't remove
-    msp: permMap.entryMicroService.pass,
+    msp: permMap.entryMsp.pass,
```
Comment (daskyrk) [inline shell/app/layout/pages/page-container/components/sidebar.tsx:59]: Is these perm config updated?

Comment (laojun) [reply shell/app/layout/pages/page-container/components/sidebar.tsx:59]: yes

**Pull request merged**